errors.go

```diff
@@ -62,6 +62,12 @@ func isResponseError(req *http.Request, resp *http.Response) *ResponseError {
 		return nil
 	}
 
+	// 472 & 473 are returned by (performance/secondary) standby instances for /sys/health requests and should
+	// not be treated as an error
+	if (resp.StatusCode == 472 || resp.StatusCode == 473) && req.URL.Path == "/v1/sys/health" {
```

Comment:
I did not see `472` mentioned in the API docs. Do you know when this gets returned?

Reply:
https://github.com/hashicorp/vault/blob/3acfa450c2605226418c7dad00b39cbfb857f7b4/website/content/api-docs/system/health.mdx#L27

It happened on my environment

Reply:
```
> GET /v1/sys/health HTTP/2
> Host: vault.co.nz:8200
> user-agent: curl/7.75.0
> accept: */*
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* Connection state changed (MAX_CONCURRENT_STREAMS == 250)!
< HTTP/2 472 
< cache-control: no-store
< content-type: application/json
< strict-transport-security: max-age=31536000; includeSubDomains
< content-length: 413
< date: Fri, 28 Jul 2023 03:51:59 GMT
<
{"initialized":true,"sealed":false,"standby":false,"performance_standby":false,"replication_performance_mode":"disabled","replication_dr_mode":"secondary","server_time_utc":1690516319,"version":"1.13.4+ent.hsm","cluster_name":"vault-cluster-4ee93d7f","cluster_id":"xx-951b-3285-d319-xxx","last_wal":10760174,"license":{"state":"autoloaded","expiry_time":"2029-01-04T00:00:00Z","terminated":false}}    
* Connection #0 to host vault.nz left intact
```

Reply:
```
> GET /v1/sys/health HTTP/2
> Host: vault.co.nz:8200
> user-agent: curl/7.75.0
> accept: */*
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* Connection state changed (MAX_CONCURRENT_STREAMS == 250)!
< HTTP/2 473
< cache-control: no-store
< content-type: application/json
< strict-transport-security: max-age=31536000; includeSubDomains
< content-length: 389
< date: Fri, 28 Jul 2023 03:54:58 GMT
<
{"initialized":true,"sealed":false,"standby":true,"performance_standby":true,"replication_performance_mode":"disabled","replication_dr_mode":"primary","server_time_utc":1690516498,"version":"1.13.4+ent.hsm","cluster_name":"vault-cluster-c08cbadf","cluster_id":"aaf2ceb9-c0fc-9691-e877-xxx","license":{"state":"autoloaded","expiry_time":"2029-07-01T00:00:00Z","terminated":false}}
* Connection #0 to host vault.co.nz left intact
```

Reply:
it's vault enterprise hsm version